src/System.Memory/src/System/Buffers/ReadOnlySequence_helpers.cs

```diff
@@ -226,24 +226,21 @@ private static void CheckEndReachable(object startSegment, object endSegment)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         private long GetLength(in SequencePosition start, in SequencePosition end)
```

Comment:
Can we remove the parameters, and just use _sequenceStart and _sequenceEnd?

Reply:
Yes. But for example if it implemented so then for example bounds checking can be implemented in one line `if (GetLength(p1, p2) < 0) throw Ex;`. I think it can be usefully in such way.

Reply:
But we don't use it in any places like that, today. And if we are allowing arbitrary SequencePositions as input, we would need to add extra checks.

Reply:
> But we don't use it in any places like that, today. And if we are allowing arbitrary SequencePositions as input, we would need to add extra checks.

Yes it does not used because we fully optimized each method. Extra checks would not needed. I can remove but what is sense?